tests/PySys/mqtt_port/mqtt_port_change_connection_works/run.py

```diff
@@ -93,6 +100,34 @@ def validate_tedge_mqtt(self):
         self.assertGrep(
             "mqtt_sub.out", "{ \"temperature\": 25 }", contains=True)
 
+    def check_if_sub_logged(self):
+        fout = Path(self.output + '/mqtt_sub.out')
+        ferr = Path(self.output + '/mqtt_sub.err')
+        n = 0
+        while n < 10:
+            if fout.is_file() or ferr.is_file():
+                return;
+            else:
+                time.sleep(1)
+                n += 1
+        self.assertFalse(True, abortOnError=True, assertMessage=None)        
+        
+    # Starting subscriber takes sometime, so instead of sleeping before moving
+    # to next operation, its better to query active connections
+    def start_subscriber(self):
+        num_clients_b4 = subprocess.getoutput("mosquitto_sub -p 8889 -t '$SYS/broker/clients/active' -C 1")
```

Comment:
Info: `tedge mqtt sub '$SYS/broker/clients/active'` is working fine ;-)

Reply:
It works only with your changes to the tedge, not with the old one!.

Reply:
Good trick, I haven't thought of this. On the drawback side, now we need to wait for 10s. The process is probably up and running far earlier :)

Reply:
Right, we have to wait for the mosquito statistics to get updated. Its not instant, it takes some time.